### PR TITLE
XP Tracker: Made bar charts lineup with each other. Added better tooltips and made X-Axis labels visible

### PR DESCRIPTION
**Pull request by eioo**
Solution for #174 .
Bit hacky but I think it's better than before. Hovering over the first column does not show tooltip.

example

### CODE REVIEW

File: src/routes/xp-show.js [hunk 1]
```diff
@@ -72,6 +72,40 @@ const createDateRange = (start, end) => {
...
+        transform="rotate(-90)"
+      >
+        {payload.value}
+      </text>
+    </g>
+  )
+}
+
+const CustomToolTip = ({ active, payload, label, description }) => {
+  console.log(payload)
```
Comment (arthurtyukayev) [inline src/routes/xp-show.js]: You've got a console log here. Please remove.

Comment (eioo) [reply src/routes/xp-show.js]: Oops, my bad. Fixed.

**Comment by deathbeam** [comment]
needs rebase

File: package-lock.json [hunk 2]
```diff
@@ -3322,8 +3322,7 @@
             "minimist": {
```
Comment (deathbeam) [inline package-lock.json]: revert changes in this file

File: src/routes/xp-show.js [hunk 3]
```diff
@@ -66,6 +66,38 @@ const createDateRange = (start, end) => {
   }
 }
 
+const RotatedXAxisTick = ({ x, y, payload }) => {
+  return (
```
Comment (deathbeam) [inline src/routes/xp-show.js]: remove return and braces, use implicit one

**Comment by deathbeam** [comment]
Also, with vertical labels the Total ranks gained is overlapping Total experience gained

**Comment by Nightfirecat** [comment]
We've removed xp tracking in favor of directing users to Wise Old Man, so this PR is no longer needed.

**Pull request closed without merging**